### PR TITLE
Upgrade to PowerShell 5.0 minimum dependency

### PR DESCRIPTION
**Pull request by maddieclayton**
## Description
https://github.com/Azure/azure-powershell/issues/5897

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by cormacpayne** [comment]
@maddieclayton looks like the build is failing because of the [`LocationCompleter`](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Websites/Commands.Websites/Cmdlets/WebApps/NewAzureWebApp.cs#L67) in `New-AzureRmWebApp`

File: src/ResourceManager/RedisCache/Commands.RedisCache.Test/Commands.RedisCache.Test.csproj [hunk 1]
```diff
@@ -213,5 +209,5 @@
   </ItemGroup>
   <ItemGroup />
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <Import Project="..\..\..\..\tools\Common.Dependencies.targets" />
+  <Import Project="..\..\..\..\tools\Common.Dependencies.Test.targets" />
```
Comment (cormacpayne) [inline src/ResourceManager/RedisCache/Commands.RedisCache.Test/Commands.RedisCache.Test.csproj]: @maddieclayton I would check to make sure the assemblies in `Common.Dependencies.targets` are included in this `.csproj` file.

**Comment by maddieclayton** [comment]
sign job here: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/409/

**Comment by maddieclayton** [comment]
@azuresdkci Test this please

**Comment by maddieclayton** [comment]
on demand here: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/powershell-demand/737/

**Pull request merged**